nc/source/api_implementation/NcCamera.cpp

```diff
@@ -0,0 +1,17 @@
+#include "NcCamera.h"
+#include "camera/MainCamera.h"
+#include "NcDebug.h"
+
+namespace nc::camera
+{
+    void NcRegisterMainCamera(Camera * camera)
```

Comment:
This is truly my dream come true. It is now SO much easier to understand at a glance what these do.